src/table/table.scss

```diff
@@ -161,6 +161,20 @@
       visibility: visible;
     }
 
+    &.iui-expanded-content {
+      transform: scaleY(0);
+      transform-origin: top;
+      transition: transform $iui-speed-fast ease-in-out;
+      overflow: hidden;
+      opacity: 0;
```

Comment:
This approach will not work.

This is what the DOM currently looks like:
![table](https://user-images.githubusercontent.com/9084735/123463042-0c7b3800-d5b9-11eb-97e6-1349b11ceae6.gif)

I'm making some changes in React that should give you classes for animating.

Reply:
We collaborated and are now using the correct classes.